### PR TITLE
TEST-325 Sync with upstream

### PR DESCRIPTION
**Pull request by Cousjava**

### CODE REVIEW

File: ejb/remote/vendor/pom.xml [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 <!-- Copyright Payara Services Limited -->
 
+<<<<<<< HEAD
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"> <modelVersion>4.0.0</modelVersion>
```
Comment (arjantijms) [inline ejb/remote/vendor/pom.xml]: This should probably not be there

**Comment by MarkWareham** [comment]
Needs a local test to confirm all is good before merge

**Comment by Pandrex247** [comment]
Tested locally, only failure was `[INFO] Java EE 7 Sample: ejb - remote - vendor - Payara and GlassFish Remote EJB Provider`, which I believe fails on the current master branch as-is.

**Comment by arjantijms** [comment]
>Tested locally, only failure was [INFO] Java EE 7 Sample: ejb - remote - vendor - Payara and GlassFish Remote EJB Provider, which I believe fails on the current master branch as-is

Which master would that be? Upstream master doesn't fail on that one.

**Comment by Pandrex247** [comment]
... this master ...
Upstream master is a different repo - that's like merging PRs against Payara because GlassFish builds :P

**Pull request merged**